src/VoltageCalibration.cc

```diff
@@ -36,10 +39,10 @@ static double* adcTablePerSample(int order, int npoints, const double * par, con
   return adcTable;
 }
 
-static double adcToVolt(double in_adc, int npoints, const double * resid_volt, const double * resid_adc)
+static double adcToVolt(double in_adc, int npoints, const double * voltTable, const double * adcTable)
 {
-  const double *volt_array = resid_volt;
-  const double *adc_array = resid_adc;
+  const double *volt_array = voltTable;
+  const double *adc_array = adcTable;
```

Comment:
why is that necessary?

Reply:
They are not necessary now, I remember that there were something wrong when I was first writing those functions so I made the local copies, but then when I fixed the major issue (probably had nothing to do with the copies, but I forgot what it was) I didn't remove them and I just kept them there. I've just removed these local copies in the main branch directly.